### ISSUE TITLE
In 1.5.0.beta5 on attempt to restore from backup nothing happens

### ISSUE THREAD

**Issue by ipk01** [xbrowsersync/app#85]
### Steps to reproduce
1. Choose "restore"
2. Paste content of .json backup file
3.

### Expected behaviour
Shouls restore bookmarks from backup

### Actual behaviour
Just hangs

### Client app information
Chrome

**App Platform Version:** 
 Version 73.0.3683.103 (Offizieller Build) (64-Bit)

#### Other info
```
Enter any other information you think would be helpful here.
```

**Comment by ipk01** [comment]
Happens on Chrome and Firefox, both unter Windows and in Ubuntu, i.e. 4 Platforms tried alltogether.

**Comment by nero120** [comment]
I'll need a bit more information to go on that that, can you please download the log covering the time when the issue occurs and paste it into this issue?

**Comment by ipk01** [comment]
From just now (I backuped and then attempted to restore from the same backup).
20190425084235236	[trace]	Starting up
20190425084235243	[trace]	{"displayPermissions":true,"appVersion":"1.5.0beta5","platform":{"name":"chrome","version":"73.0.3683","mobile":false,"os":"Windows 10"}}
20190425084250203	[trace]	Optional permissions granted
Does not seem very useful ...

**Comment by nero120** [comment]
I've fixed a bug that could be the cause of your issue, please could you download [v1.5.0beta6](https://github.com/xbrowsersync/app/releases/tag/v1.5.0beta) and try again?

If the problem occurs again please paste the log as above.

**Comment by ipk01** [comment]
Problem solved with v1.5.0beta6, Thanks!
